nginx/tests/fixtures/plus_api_http_location_zones.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Is it possible to reorganize this `fixtures/*` folder with `fixtures/v6/*`, `fixtures/v7/*`

Reply:
Updated!